runbot_language/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- encoding: utf-8 -*-
+##############################################################
+#    Module Writen For Odoo, Open Source Management Solution
+#
+#    Copyright (c) 2011 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+#    coded by: moylop260@vauxoo.com
+############################################################################
+
+{
+    'name': 'Runbot Language',
+    'category': 'Website',
+    'summary': 'Runbot',
+    'version': '1.1',
+    "website": "http://www.vauxoo.com/",
+    'description': "Runbot with posibility to indicate the language"
+                   " in the instances generated",
+    'author': 'Vauxoo',
```

Comment:
Could you use the [README.rst instead of description and icon template](https://github.com/OCA/maintainer-tools/tree/master/template/module) and add `",Odoo Community Association (OCA)"` to author